clang/include/clang/Format/Format.h

```diff
@@ -2421,20 +2421,28 @@ struct FormatStyle {
 
   /// Indent goto labels.
   ///
-  /// When ``false``, goto labels are flushed left.
   /// \code
-  ///    true:                                  false:
-  ///    int f() {                      vs.     int f() {
-  ///      if (foo()) {                           if (foo()) {
-  ///      label1:                              label1:
-  ///        bar();                                 bar();
-  ///      }                                      }
-  ///    label2:                                label2:
-  ///      return 1;                              return 1;
-  ///    }                                      }
+  ///    GLI_Indent:                            GLI_None:
+  /// namespace NS {                  vs.     namespace NS {
+  ///   class C {                              class C {
+  ///     void foo() {                           void foo() {
+  ///       while (x) {                            while (x) {
+  ///         if (y) {                               if (y) {
+  ///         label:                               label:
```

Comment:
This does not display what `None` should do. You would break thousands of users.

Reply:
looking into this